### PR TITLE
FEAT: Add Tools Support for Qwen Series MOE Models

### PR DESCRIPTION
**Pull request by zhanghx0905**
This PR adds tools support for the Qwen series Mixture of Experts (MOE) models. The Qwen Chat series MOE models possess similar tool call capabilities as other dense models, enhancing their utility and consistency.

### CODE REVIEW

File: xinference/model/llm/llm_family.json [hunk 1]
```diff
@@ -2290,7 +2290,8 @@
       "zh"
     ],
     "model_ability": [
-      "chat"
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:2293]: Can you add the tools to `llm_family_modelscope.json`?

**Comment by zhanghx0905** [comment]
Also, the size of the qwen MOE model is incorrectly labeled; the actual number of parameters for Qwen1.5-MoE-A2.7B-Chat is 14B, whereas the actual number of parameters for Qwen2-57B-A14B is 57B.

**Comment by qinxuye** [comment]
> Also, the size of the qwen MOE model is incorrectly labeled; the actual number of parameters for Qwen1.5-MoE-A2.7B-Chat is 14B, whereas the actual number of parameters for Qwen2-57B-A14B is 57B.

That depends on how the size is defined, we’d prefer to treat it as activated parameters size.

**Comment by zhanghx0905** [comment]
> > Also, the size of the qwen MOE model is incorrectly labeled; the actual number of parameters for Qwen1.5-MoE-A2.7B-Chat is 14B, whereas the actual number of parameters for Qwen2-57B-A14B is 57B.
> 
> That depends on how the size is defined, we’d prefer to treat it as activated parameters size.

In this case, model memory calculation will also conduct according to the number of active parameters. The memory estimation for MOE models will be wrong.

**Comment by qinxuye** [comment]
Looks like #1648 added moe in it as well, can you look at it?

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**